client/package.json

```diff
@@ -19,6 +19,7 @@
     "react-dom": "^17.0.2",
     "react-dropzone": "^10.2.1",
     "react-google-recaptcha": "^2.1.0",
+    "react-query": "^3.34.15",
```

Comment:
This is just a question 😆
Are we now allowed to use this for queries where state is handled by server or a very local state?
Would there be a guideline for when to use this vs boilerplated redux for newer queries?